tests/python/gaia-ui-tests/gaiatest/tests/functional/ftu/test_ftu_status_bar.py

```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.ftu.app import Ftu
+from gaiatest.apps.system.regions.status_bar import StatusBar
+
+
+class TestFtu(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.ftu = Ftu(self.marionette)
```

Comment:
I think it's better if we're consistent about when we launch the app. I would move it to the test itself like we do in the other tests.

By the way, to me it would be better to run the app in setUp(). Because it sounds like a pre-requisite to run the test. It might be a good refactor task. What do you think?